app/controllers/api/v2/models_controller.rb

```diff
@@ -47,6 +47,12 @@ def update
       def destroy
         process_response @model.destroy
       end
+
+      private
+
+      def find_resource
+        @model = Model.friendly.find(params[:id]) || super
+      end
     end
```

Comment:
IIRC this is here because some models could have names like `101` that if we use `from_param` or `friendly` search first, we might return the wrong Model, @unorthodoxgeek ?